File: ytube-extractor/src/main/java/com/raka/ytube_extractor/models/youtube/playerResponse/Title.java
package com.raka.ytube_extractor.models.youtube.playerResponse;

import java.io.Serializable;

public class Title implements Serializable {
    private String simpleText;

    public String getSimpleText() {
        return simpleText;
    }

    public void setSimpleText(String simpleText) {
        this.simpleText = simpleText;
    }

    @Override
    public String toString() {
        return
                "Title{" +
                        "simpleText = '" + simpleText + '\'' +
                        "}";
    }
}

File: ytube-extractor/src/main/java/com/raka/ytube_extractor/models/newModels/Title.kt
package com.raka.ytube_extractor.models.newModels

import com.google.gson.annotations.SerializedName

class Title {
    @SerializedName("simpleText")
    var simpleText: String? = null
    override fun toString(): String {
        return "Title{" +
                "simpleText = '" + simpleText + '\'' +
                "}"
    }
}